src/app/metadata/features/metadata-browser/metadata-browser.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="grid grid-cols-12 grid-rows-none gap-x-5">
-  <div class="col-span-9 col-start-4">
+  <div class="col-span-12 xl:col-span-9 xl:col-start-4">
```

Comment:
Maybe use the sidebar also with lg not just xl (but I'm not sure and let you decide).